packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -255,7 +255,9 @@ export function LineChart(props: LineChartProps): ReactElement | null {
     );
 }
 
-function getDataTypes(series: LineChartSeries[]): { x: string; y: string } | Error | undefined {
+type DataTypeResult = { x: string; y: string };
+
+function getDataTypes(series: LineChartSeries[]): DataTypeResult | Error | undefined {
     let dataTypes: { x: string; y: string } | undefined;
```

Comment:
Here you can also use `DataTypeResult | undefined` i think

Reply:
Good one! Changed it.